graphics/src/MeshManager.cc

```diff
@@ -146,19 +146,23 @@ const Mesh *MeshManager::Load(const std::string &_filename)
     MeshLoader *loader = nullptr;
 
     loader = &this->dataPtr->assimpLoader;
-    /*
+
     if (extension == "stl" || extension == "stlb" || extension == "stla")
       loader = &this->dataPtr->stlLoader;
     else if (extension == "dae")
       loader = &this->dataPtr->colladaLoader;
     else if (extension == "obj")
-      loader = &this->dataPtr->objLoader;
+      //loader = &this->dataPtr->objLoader;
+      loader = &this->dataPtr->assimpLoader;
+    else if (extension == "gltf")
+      loader = &this->dataPtr->assimpLoader;
+    else if (extension == "fbx")
+      loader = &this->dataPtr->assimpLoader;
```

Comment:
Any specific reason for this change? My hope (if possible!) was to be able to substitute all the loaders with the assimp loader and I already got most static DAE assets, as well as some animated ones, to work fine. We can fallback later to the other loaders if we find an issue that can't be handled by assimp if that makes sense?

Reply:
Yes sure, should I git force push to remove the commit, or does GitHub gives you option to select commits to merge?

Reply:
I can't really do that it's either merge all or nothing!

Reply:
Okay, gonna fix it later today.

Reply:
@luca-della-vedova  fixed.